### DISCUSSION TITLE
Helm charts removed?

### DISCUSSION THREAD

**Discussion by majolo** [minio/minio#13759] [Q&A] [answered]
Hi, I've noticed https://github.com/minio/charts and https://helm.min.io/ now 404. 

Have these been recently removed and what is the reason for their removal?

Thanks in advance,
Majolo

**Comment by majolo** [comment]
I assume they've been replaced by: https://github.com/minio/minio/tree/master/helm/minio

Is there a reason the repo was deleted and not deprecated + archived?

**Comment by RafalSkolasinski** [comment]
This has broken our pipeline. Also it seems that the charts from new location are not a drop-in replacement as they seem to for some reason create many too many pods 
![image](https://user-images.githubusercontent.com/10928117/143444731-be56057a-16b8-4fb8-a4be-694a04a37453.png)

~~and there is also only one version published there~~ (actually there is more, my bad when checking)
```
minio/minio  	3.2.0        	RELEASE.2021-10-13T00-23-17Z	Multi-Cloud Object Storage                        
```

**Reply by larsla** [reply]
Default number of replicas is set to `16` for some reason:
https://github.com/minio/minio/blob/a5db4ca09277ced1b2488b437c0d63eb3de4147d/helm/minio/values.yaml#L102

**Reply by RafalSkolasinski** [reply]
not only that, I had to additionally add
```
mode: standalone
```
for it to starts behaving more like before

I do still however struggle to expose UI under `http(s)://{ some ingress or ip}/minio/` (used to work without issues...)

**Reply by larsla** [reply]
The `defaultBucket` key changed to `buckets`:
```yaml
buckets:
  - name: myBucket
```

And `accessKey`/`secretKey` changed to `rootUser`/`rootPassword`:
```yaml
rootUser: myUser
rootPassword: mySecret
```

With these values I have it working again.
I did lose my storage in the confusion, don't know if it would have been avoided if the upgrade worked on the first try or not. Will need to investigate that more.

**Reply by paolomainardi** [reply]
Frankly this is quite a mess and very disappointed about all of this, we are going to migrate away from this chart, it is totally unreliable.

**Reply by RafalSkolasinski** [reply]
We have exactly the same feeling. Most of today lost for me to trying to figure this out. 
Lack of communication and just downgrading a valid issue to discussion without any comment from maintainers is ~very~ slightly disappointing.

**Reply by harshavardhana** [reply]
The issue was on our end, doing some cleanup the charts repo was deleted, unfortunately. It is restored back here https://github.com/minio/charts

**Reply by RafalSkolasinski** [reply]
Nice, thanks for bringing it back!

**Comment by harshavardhana** [comment, marked as answer]
https://github.com/minio/charts - this is restored and it was removed as a mistake.

**Reply by paolomainardi** [reply]
thanks @harshavardhana, should we migrate to the other one ? is it the new official one ?

**Reply by harshavardhana** [reply]
@paolomainardi well migration is not simple since they are not in sync yet and do not do the same things. So use the way are you using we are not yet done with syncing the new vanilla with the older helm chart.

**Reply by majolo** [reply]
Thanks @harshavardhana!

**Comment by Linaa** [comment]
old helm chart has been removed again it seems!  can we put it back?
https://helm.min.io/ returns 404.

**Reply by harshavardhana** [reply]
No @Linaa its 2yrs since its deprecated, time it move to newer repo or operator.

**Reply by yanxon** [reply]
@harshavardhana, how can we access it now?

**Reply by harshavardhana** [reply]
You cannot and you should have migrated @yanxon - it is a deprecated project for 2yrs+

**Reply by yanxon** [reply]
@harshavardhana, thank you for the reply.

If I understand you correctly, minio can no longer be downloaded/installed through helm. If this is the case, may I ask where does MinIO migrate to? Is there a documentation for this migration?

**Reply by harshavardhana** [reply]
Again, I repeat, it has been 2yrs since that has been deprecated. 

For those living under the rock

- https://github.com/minio/operator/tree/master/helm/operator and https://github.com/minio/operator/tree/master/helm/tenant supported and recommend by MinIO

- https://github.com/minio/minio/tree/master/helm/minio (vanilla helm charts without operator, community support only)

**Reply by RafalSkolasinski** [reply]
Don't fix what ain't broken, it worked fine since now :)
